fuse_models/include/fuse_models/parameters/acceleration_2d_params.h

```diff
@@ -68,6 +68,9 @@ struct Acceleration2DParams : public ParameterBase
 
       nh.getParam("disable_checks", disable_checks);
       nh.getParam("queue_size", queue_size);
+
+      throttle_period.fromSec(fuse_core::getPositiveParam(nh, "throttle_period", throttle_period.toSec(), false));
```

Comment:
@svwilliams I just fixed this because I was using `getPositiveParam` incorrectly. I thought it was "returning" by argument. :man_facepalming: 

Any comments regarding this way to call it (maybe it's cleaner in two lines with a local variable)? and whether the `getPositiveParam` API is confusing being the only one that returns the param instead of changing the input/output argument (as `getParam` and `getParamRequired`)?

Reply:
No particular reason, really. I'm sure I just wasn't thinking about the standard ROS function signatures at the time. I'll gladly merge a PR that changes the signature, if you get to it before I do.

Reply:
https://github.com/locusrobotics/fuse/pull/169